clinica/pydra/engine.py

```diff
@@ -19,8 +20,9 @@ def run_wrapper(
 
         pipeline = Workflow(
             name=name,
-            input_spec=["input_dir"],
-            input_dir=input_dir,
+            input_spec=["bids_dir", "caps_dir"],
+            bids_dir=input_dir,
```

Comment:
I suppose it's better to keep `bids_dir` instead of `input_dir` as the inputs also come from the CAPS